examples/java_keystore/BUILD.bazel

```diff
@@ -0,0 +1,37 @@
+load("//distroless:defs.bzl", "java_keystore")
+load("//distroless/tests:asserts.bzl", "assert_jks_listing", "assert_tar_listing")
+
+java_keystore(
+    name = "java_keystore",
+    certificates = [
+        # asserting that we support both bundle x509 certs
+        # and single x509 certs
+        "amazon.crt",
```

Comment:
I know this doesn't matter, but it does feel a bit strange to have the amazon cert in here.

Reply:
i took it from https://github.com/GoogleContainerTools/distroless/blob/be23a86353166b733a7088a35fa357cc79afbe60/cacerts/jksutil/test.crt#L4 and split the amazon ca-cert out of bundle to test both cases.

Reply:
@loosebazooka wdyt?

Reply:
oh lulz, sgtm